fairtally/cli.py

```diff
@@ -50,3 +50,4 @@ def cli(urls, input_file, output_format, output_filename):
 
     msg = f"Completed fairtally on {len(all_urls)} URLs and written report to {output_filename}"
     current_value.set_description_str(msg)
+    current_value.close()
```

Comment:
Other way
```suggestion
    current_value.set_description_str((len("currently checking ") + max([len(u) for u in all_urls])) * ' ')
    current_value.close()
    click.echo(msg, err=True)
```

It leaves an empty line
```
(env) verhoes@slurper ~/git/fair-software/fairtally $ fairtally https://github.com/fair-software/fairtally
fairtally progress: |████████████████████████████████████████████| 1/1
                                                             
Completed fairtally on 1 URLs and written report to tally.html
(env) verhoes@slurper ~/git/fair-software/fairtally $ 
```
The last url checked is not visible anymore and no lingering text on next prompt.